Assets/Scripts/UI/Text/TextController.cs

```diff
@@ -5,16 +5,28 @@
 
 public class TextController : MonoBehaviour
 {
+    public int velocityMultiplier = 10;
+
     public Text timerText;
     public Text positionText;
     public Text speedText;
 
+    private int playerIndex;
+
     public enum TextTypes{ timer, position, speed};
 
     // Start is called before the first frame update
     void Start()
     {
         ResetTexts();
+        playerIndex = GetPlayerIndex();
+    }
+
+    public void Update()
+    {
+        SetText(TextTypes.position, RacerManager.GetPositionTracker(playerIndex).GetPosition().ToString());
```

Comment:
Upon further investigation on the index out of range exception throw it seems that this line keeps trying to access index 3 of the position tracker list even when a racer is removed from the list which causes the exception. Please fix